runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/CacheFetchIT.java

```diff
@@ -451,6 +453,22 @@ public void shouldReceiveMessagesWithNoFilter() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("cache.options.live.yaml")
+    @Specification({
+        "${app}/messages.after.retention.max/client",
+        "${app}/messages.after.retention.max/server"})
+    @ScriptProperty("serverAddress \"zilla://streams/app1\"")
+    @Configure(name = KafkaConfigurationTest.KAFKA_CACHE_RETENTION_MILLIS_MAX_NAME, value = "1000")
+    public void shouldReceiveMessagesAfterRetentionMax() throws Exception
+    {
+        partition.append(10L);
+        k3po.start();
+        Thread.sleep(1500);
```

Comment:
Lets drop this down a good bit, like `200ms` and `250ms` perhaps to avoid unnecessary build pause.